### DISCUSSION TITLE
RLS Policy to restrict column user from adding a row to another table

### DISCUSSION THREAD

**Discussion by NielsVeen** [supabase/supabase#15447] [Questions]
I'm currently building an auctioning website but I can't seem to figure out the right RLS policy to restrict a user that has created a car from bidding on the same car. My Policy currently looks like this:

`BEGIN;
  ALTER POLICY "Don't allow owners" ON "public"."bids" WITH CHECK ((true AND (EXISTS ( SELECT 1 FROM cars WHERE ((cars.car_id = bids.car_id) AND (cars.user_id = auth.uid()))))));
COMMIT;`

Any tips on how to make this work? Or should I just not use a RLS policy and use a trigger?

**Comment by GaryAustin1** [comment]
How does it "not work"?
Also you will need to meet RLS on cars select policy (my guess is you do).
